tests/local/delta/test_DeltaTableSpec.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+from textwrap import dedent
+
+from pyspark.sql import types as t
+
+from spetlr import Configurator
+from spetlr.deltaspec.DeltaTableSpec import DeltaTableSpec
+from tests.cluster.delta.deltaspec import tables
+
+
+class TestDeltaTableSpec(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        c = Configurator()
+        c.clear_all_configurations()
+        c.set_prod()
+
+        cls.base = tables.base
+        cls.target = tables.target
+
+    def test_01_create_statements(self):
+        self.assertEqual(
+            self.base.get_sql_create(),
+            dedent(
+                """\
+                CREATE TABLE myDeltaTableSpecTestDb{ID}.tbl
+                (
+                  c double,
+                  d string COMMENT "Whatsupp",
+                  onlyb int,
+                  a int,
+                  b string
+                )
+                USING DELTA
+                LOCATION "dbfs:/tmp/somewhere{ID}/over/the/rainbow"
+                TBLPROPERTIES (
+                  "delta.columnMapping.mode" = "name",
+                  "delta.minReaderVersion" = "2",
+                  "delta.minWriterVersion" = "5"
+                )
+                """
+            ),
+        )
+
+    def test_01_diff_alter_statements(self):
```

Comment:
do you test if the table needs a new location?

Reply:
added a test